lz4/stream/__init__.py

```diff
@@ -12,7 +12,9 @@ class LZ4StreamDecompressor:
     """ LZ4 stream decompression context.
 
     """
-    def __init__(self, strategy, buffer_size, return_bytearray=False, store_comp_size=4, dictionary=""):
+
+    def __init__(self, strategy: str, buffer_size: int, return_bytearray: bool = False, store_comp_size: int = 4,
+                 dictionary: Union[str, bytes, memoryview] = ""):
```

Comment:
```suggestion
                 dictionary: BytesLike = ""):
```
And up in the file:
```
BytesLike = Union[str, bytes, memoryview]
```

Apply to the others too, this makes it easier to not f-up when writing the same union types